02_configure_host.sh

```diff
@@ -128,11 +135,7 @@ if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
   if [ "$ADDN_DNS" ] ; then
     echo "server=$ADDN_DNS" | sudo tee /etc/NetworkManager/dnsmasq.d/upstream.conf
```

Comment:
Only for CentOS

Reply:
Yeah. I will update it.
Btw, do we need to configure dnsmasq, since I see that they have a dnsmasq running on a container ?